vendor/cookbooks/rails/templates/default/app_passenger_nginx.conf.erb

```diff
@@ -25,6 +25,9 @@ server {
 
   passenger_enabled on;
   passenger_app_env <%= @rails_env %>;
+
+  client_max_body_size <%= @client_max_body_size %>;
```

Comment:
At this point we don't provide a default right? So when a person does not provide a `max_body_size` the config becomes invalid?